docs/how-to/http-proxy.md

```diff
@@ -0,0 +1,31 @@
+# How to run Piraeus when using a HTTP proxy (like Squid)
+
+When attempting to deploy Piraeus behind a HTTP proxy, the following configuration needs to be done.
```

Comment:
Our guides usually start with a small introduction like:

> This guide shows you how to ... 
> 
> To complete this guide, you should be familiar with: ...

See https://github.com/piraeusdatastore/piraeus-operator/blob/v2/docs/how-to/talos.md for examples.

Reply:
Changed accordingly